docs/cookbook/promotions/catalog-promotion-priority.rst

```diff
@@ -0,0 +1,20 @@
+How to manage catalog promotion priority?
```

Comment:
I wonder if the cookbook is a proper place for this explanation 🤔 I would be for the catalog promotion section in the book: https://docs.sylius.com/en/latest/book/products/catalog_promotions.html